### PR TITLE
Adds tox/molecule test 

### PR DESCRIPTION
**Pull request by ahogui12**
https://jenkins.wazo.community/job/ansible-role-metadata/

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+pipeline {
+  agent {
+    label 'molecule'
+  }
+  stages {
+    stage ('Test') {
+      environment {
+        WAZO_MOLECULE_LOGS_DIR = "${WORKSPACE}/molecule/logs"
```
Comment (sduthil) [inline Jenkinsfile]: Is this variable really used?

File: molecule/default/molecule.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+dependency:
+  name: galaxy
+  options:
+    ignore-certs: True
```
Comment (sduthil) [inline molecule/default/molecule.yml]: We should not ignore certificate errors when using Galaxy. Is this necessary?

**Review by sduthil — DISMISSED** [review]

**Comment by fblackburn1** [comment]
recheck

**Review by fblackburn1 — APPROVED** [review]

**Comment by fblackburn1** [comment]
regate

**Pull request merged**